spinta/types/type.py

```diff
@@ -18,63 +20,84 @@ class Type:
         'link': {'type': 'string'},
     }
 
-    def is_valid(self, value):
-        raise NotImplementedError
+    def load(self, value: typing.Any):
+        # loads value given by the user to native Python type
+        # this is done before validation and writing to database
+        raise NotImplementedError(f"{self.__class__} lacks implementation for load()")
+
+    def is_valid(self, value: typing.Any):
+        # checks if value is valid for the use (i.e. valid range, etc.)
+        raise NotImplementedError(f"{self.__class__} lacks implementation for is_valid()")
+
+    def prepare_for_mongo(self, value: typing.Any):
+        # prepares value for Mongo store
+        # for simple types - loaded native values should work
+        # otherwise - override for this method is necessary
+        return value
+
+    def prepare_for_postgres(self, value: typing.Any):
+        # prepares value for PostgreSQL store
+        # for simple types - loaded native values should work
+        # otherwise - override for this method is necessary
+        return value
```

Comment:
These two methods should be replaced by commands and should go to `spinta/backends/...`.